.github/workflows/container.yml

```diff
@@ -72,6 +74,9 @@ jobs:
     with:
       pr-number: ${{ github.event.number }}
   build_and_push:
+    concurrency:
```

Comment:
I want to this job with build and push, but it has many overhead for reusing the podman image